### PR TITLE
Update mailbox-usage.md

### PR DESCRIPTION
**Pull request by msbemba**
The reports show up only when you click on view more, Hence made changes to the navigation.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3247

File: microsoft-365/admin/activity-reports/mailbox-usage.md [hunk 1]
```diff
@@ -32,8 +32,7 @@ The **Mailbox usage report** provides information about users with a user mailbo
 
 1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page.
 
-    
-2. From the **Select a report** drop-down, select **Exchange** \> **Mailbox usage**.
+2. Click on **View More**, From the **Select a report** drop-down, select **Exchange** \> **Mailbox usage**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/mailbox-usage.md]:
```suggestion
2. Select **View More**.
3. From the **Select a report** drop-down, choose **Exchange** \> **Mailbox usage**.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@kwekuako Conflict resolved, please merge this PR. Thanks.

**Pull request merged**